packages/synthetix-main/contracts/interfaces/IFundSCCP.sol

```diff
@@ -0,0 +1,20 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/// @title Module for managing preferred and approved funds via SCCPs
+interface IFundSCCP {
```

Comment:
maybe we could do better for naming this file/interface

Reply:
Fixed. Changed to FundConfiguration